spec.md

```diff
@@ -3963,12 +3963,12 @@ Note that the logical consequence is that the and-reduction of a zero-width expr
 
 ## Concatenate Operation
 
-| Name | Arguments | Parameters | Arg Types    | Result Type | Result Width |
-|------|-----------|------------|--------------|-------------|--------------|
-| cat  | (e1,e2)   | ()         | (UInt, UInt) | UInt        | w~e1~+w~e2~  |
-|      |           |            | (SInt, SInt) | UInt        | w~e1~+w~e2~  |
+| Name | Arguments | Parameters | Arg Types | Result Type | Result Width |
+|------|-----------|------------|-----------|-------------|--------------|
+| cat  | (e\*)     | ()         | (UInt\*)  | UInt        | sum(w\~e\*)  |
+|      |           |            | (SInt\*)  | UInt        | sum(w\~e\*)  |
 
-The result of the concatenate operation is the bits of e1 concatenated to the most significant end of the bits of e2.
+The result of the concatenate operation is the bits of all input expressions concatenated together, with the first argument placed at the most significant end and subsequent arguments concatenated toward the least significant end.
```

Comment:
Maybe there is a place to indicate that `*` means zero or more repetition. Also, it would be good to mention in prose that concatenate can take zero arguments and will produce a zero width result.